bouter/embedded/__init__.py

```diff
@@ -6,90 +6,91 @@
 
 
 class EmbeddedExperiment(Experiment):
-    def __init__(self, *args, continue_curvature=None, **kwargs):
```

Comment:
initial processing is removed? proabably makes sense

Reply:
yup, will have to be explicit